<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StreamingSettingsPage</class>
 <widget class="QWidget" name="StreamingSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Streaming</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QCheckBox" name="use_streaming">
     <property name="text">
      <string>Active steaming</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>This enables HTTP streaming. A client can stream any Clementine URL except for Spotify.</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="use_stream_container_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Settings</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="label_stream_port">
        <property name="minimumSize">
         <size>
          <width>171</width>
          <height>0</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Port</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="stream_port">
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>8080</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Bitrate</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QSlider" name="bitrate_slider">
          <property name="maximum">
           <number>250</number>
          </property>
          <property name="value">
           <number>128</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="bitrate_box">
          <property name="suffix">
           <string> kBps</string>
          </property>
          <property name="maximum">
           <number>250</number>
          </property>
          <property name="value">
           <number>128</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>use_streaming</sender>
   <signal>toggled(bool)</signal>
   <receiver>use_stream_container_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>57</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bitrate_box</sender>
   <signal>valueChanged(int)</signal>
   <receiver>bitrate_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bitrate_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>bitrate_box</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
